src/main/constraints/95_async_tasks.sql

```diff
@@ -0,0 +1,11 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE async_task_status
```

Comment:
I think these constraints files typically define the constraints for the table that matches the filename: https://github.com/cyverse-de/de-db#constraints

So I would expect to find this foreign key defined in some `XX_async_task_status.sql` table, and the one below defined in its own `XX_async_task_behavior.sql` file.